caravel/models.py

```diff
@@ -1059,8 +1240,78 @@ def fetch_metadata(self):
         if not self.main_dttm_col:
             self.main_dttm_col = any_date_col
 
+    def alter_json_metadata(self, **kwargs):
+        d = self.dict_metadata
+        d.update(kwargs)
+        self.json_metadata = json.dumps(d)
+
+    @property
+    def dict_metadata(self):
+        if self.json_metadata:
+            return json.loads(self.json_metadata)
+        return {}
+
+    @classmethod
+    def import_obj(cls, datasource_to_import, import_time=None):
```

Comment:
@mistercrunch - making query to retrieve the object that have remote id is not possible as `"remote_id": {id}` could be in the end of the json and there will be no comma after it. Json doesn't support trailing commas. Because of it I loop through all slices / dashboards to find the object to override.
I see 2 possible solutions: 
* use `"remote_id": {id}{prefix}`instead
* add remote_id field to the model

What would you suggest?
Would it be fine to have it in the separate PR ?